### PR TITLE
Merge 9.3 release notes to trunk

### PR DESCRIPTION
**Pull request by oguzkocer**
Also includes the `release-toolkit` update to `4.2` in 856103e75fe6791ee4f1c65ff99bfe4aee2f381e.

### CODE REVIEW

File: WooCommerce/metadata/PlayStoreStrings.pot [hunk 1]
```diff
@@ -11,20 +11,20 @@ msgstr ""
...
-msgctxt "release_note_092"
+msgctxt "release_note_093"
 msgid ""
-"9.2:\n"
-"Much of the information on an order from a customer is important! We’ve heard you’d like to copy and paste some of those details outside the app. Now you can long-press on most fields to copy them to the clipboard. Paste away in a text message, e-mail, or note!\n"
+"9.3:\n"
+"Canada, say hello to in-person payments! We are thrilled to announce that in-person payments are now available to Canadian merchants currently using WooCommerce Payments.\n"
 "\n"
-"We’ve also addressed several bugs around in-person payments and order creation. Thank you for your feedback!\n"
+"This release also contains a few minor improvments. Thank you all again for your amazing feedback. Keep it coming!\n"
```
Comment (AliSoftware) [inline WooCommerce/metadata/PlayStoreStrings.pot]: Typo:
```suggestion
"This release also contains a few minor improvements. Thank you all again for your amazing feedback. Keep it coming!\n"
```

File: WooCommerce/metadata/release_notes.txt [hunk 2]
```diff
@@ -1,6 +1,3 @@
-- [*] In-Person Payments: Individual Card Reader Manual tabs removed from IPP Screen and moved to the new screen `Card Reader Manuals` [https://github.com/woocommerce/woocommerce-android/pull/6518]
-- [*] Show "We don't support stripe in Canada" when both WCPay and Stripe are active [https://github.com/woocommerce/woocommerce-android/pull/6573]
-- [***] In-Person Payments: In-Person Payments in Canada is now available for public! [https://github.com/woocommerce/woocommerce-android/pull/6579]
-- [*] In-Person Payments: Removed unnecessary notification when interac refund is initiated [https://github.com/woocommerce/woocommerce-android/pull/6601]
-- [*] In-Person Payments: Do not connect to the last know reader automatically after logout and site change [https://github.com/woocommerce/woocommerce-android/pull/6616]
+Canada, say hello to in-person payments! We are thrilled to announce that in-person payments are now available to Canadian merchants currently using WooCommerce Payments.
 
+This release also contains a few minor improvments. Thank you all again for your amazing feedback. Keep it coming!
```
Comment (AliSoftware) [inline WooCommerce/metadata/release_notes.txt]:
```suggestion
This release also contains a few minor improvements. Thank you all again for your amazing feedback. Keep it coming!
```

**Review by AliSoftware — APPROVED** [review]
Approving to unblock, but disabled auto-merge to give you the opportunity to fix the typo I spotted before merging.

**Comment by oguzkocer** [comment]
@AliSoftware Thanks for the review and spotting the typo 🙇

**Pull request merged**